tests/distributed/test_dist_graph_store.py

```diff
@@ -583,6 +583,7 @@ def check_server_client_hetero(shared_mem, num_servers, num_clients):
 
 @unittest.skipIf(os.name == 'nt', reason='Do not support windows yet')
 @unittest.skipIf(dgl.backend.backend_name == "tensorflow", reason="TF doesn't support some of operations in DistGraph")
+@unittest.skipIf(dgl.backend.backend_name == "mxnet", reason="Turn off Mxnet support")
```

Comment:
does it not work in mxnet?

Reply:
Yes, this will fail with mxnet.

Reply:
it seems you didn't make any modification. did it fail before?

Reply:
It will hang the mxnet test.